proto/umee/ibctransfer/v1/tx.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+package umee.ibctransfer.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "cosmos/msg/v1/msg.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/ibctransfer";
+
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.messagename_all)     = true;
+
+// Msg defines the x/ibctransfer module's Msg service.
+service Msg {
+  // UpdateIBCDenomsRateLimit adds new rate limits for ibc denoms or
+  // updates the rate limits for existed ibc denoms.
+  rpc UpdateIBCDenomsRateLimit(MsgUpdateIBCDenomsRateLimit) returns (MsgUpdateIBCDenomsRateLimitResponse);
+
+  // UpdateIBCTransferPauseStatus
+  rpc UpdateIBCTransferPauseStatus(MsgUpdateIBCTransferPauseStatus) returns (MsgUpdateIBCTransferPauseStatusResponse);
+}
+
+// MsgUpdateIBCDenomsRateLimit defines the Msg/UpdateIBCDenomsRateLimit request type.
+message MsgUpdateIBCDenomsRateLimit {
+  option (gogoproto.goproto_getters)  = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (cosmos.msg.v1.signer)       = "authority";
+
+  // authority is the address of the governance account.
+  string authority   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string title       = 2;
+  string description = 3;
+  // new_ibc_denoms_rate_limits defines the rate limits for ibc denoms tokens.
+  repeated MsgRateLimit new_ibc_denoms_rate_limits = 4 [(gogoproto.nullable) = false];
+  // update_ibc_denoms_rate_limits defines the new rate limits for ibc denoms tokens.
+  repeated MsgRateLimit update_ibc_denoms_rate_limits = 5 [(gogoproto.nullable) = false];
```

Comment:
There are only two quotas. So we should change to:
```suggestion
  // total quota for all combined outflows
  string total = 4 [(cosmos_proto.scalar)  = "cosmos.Int", (gogoproto.customtype) = "Int", (gogoproto.nullable)   = false  ];
  // per_denom quota for outflows per denom. All denoms have the same quota size.
  string per_denom = 5 [(cosmos_proto.scalar)  = "cosmos.Int", (gogoproto.customtype) = "Int", (gogoproto.nullable)   = false  ];
```

Reply:
Few weeks ago we discussed to simplify the design. The idea is to have only 2 quotas: one for all combined outflows, and other one per token. But each token will have same quota.